### ISSUE TITLE
PLANNING - APFO - DUSWM review populated when it shouldn't have

### ISSUE THREAD

**Issue by AReed327** [dhenry-KCI/FredCo-Post-Go-Live-#477]
Project Number - A258895
Observed behavior - DUSWM review populated in error. W/S designation is MU (Multi Use - private)
Expected behavior - DUSWM review should not populate. (no DUSWM fee populated so that part is correct)
Screenshot -
![image](https://user-images.githubusercontent.com/58854333/76100391-905f7a00-5f9a-11ea-9d79-09c3ad6fc539.png)
![image](https://user-images.githubusercontent.com/58854333/76100415-9d7c6900-5f9a-11ea-8403-95c86c03971b.png)

**Comment by megisys** [comment]
This issue will be wide ranging as MU was not discussed previously. Watch for more issues here.

**Comment by megisys** [comment]
PLN_APFO_DUSWM_Review_AOC - Updated for MU

**Comment by MLWilkins** [comment]
Now DUSWM fee IS populating and it shouldnt when a MU/private system, and now the transportation Engineering fee Isnt population.  When water/sewer= MU private no DUSWM fee or review should populate, but Trans Eng fee and review must populate always.


![image](https://user-images.githubusercontent.com/50637067/76852467-c49c2b80-6821-11ea-9fe1-f72e336f823c.png)
 

Afgter paying the the DUSWM fee (that shouldnt have populated) the review is correct-no DUSWM and a Trans Eng review.

**Comment by megisys** [comment]
That fee listed looks at detail page for Study water and Sewer = True. Not linked to parcel details

**Comment by megisys** [comment]
The no Study fee is linked to the no study option on the detail page. Is this right ?

**Comment by AReed327** [comment]
That's correct. The No Study option on APFO worktype STANDARD produces the $104 fee. I have retested on an APFO exemption application to replicate the initial issue on project # A258895 where a DUSWM review populated in error. That has been corrected. No DUSWM review populated on A258934. AREED
![image](https://user-images.githubusercontent.com/58854333/76864060-db00b200-6836-11ea-91dc-0eb34ffc53b0.png)
![image](https://user-images.githubusercontent.com/58854333/76864107-ed7aeb80-6836-11ea-85d1-eaf5ddeb1a8f.png)
